resources/definitions/lulzbot_base.def.json

```diff
@@ -0,0 +1,179 @@
+        "first_start_actions":["MachineSettingsAction"],
+        "has_machine_quality": true,
+        "has_materials": true,
+        "has_variants":true,
+        "machine_extruder_trains": {
+            "0": "lulzbot_base_285_extruder"
+        },
+        "manufacturer": "LulzBot",
+        "preferred_variant_name": "LulzBot SE",
+        "preferred_material":"generic_pla",
+        "supported_actions": ["MachineSettingsAction"],
+        "variants_name": "Tool Head",
+        "visible": false
+    },
+    "overrides":
+    {
+        "infill_before_walls":
+        {
+            "default_value": "False"
+        },
+        "layer_height_0":
+        {
+            "maximum_value_warning": "0.5"
+        },
+        "material_print_temperature":
+        {
+            "maximum_value_warning": "295",
+            "description": "The temperature used for printing. Set at 0 to pre-heat the printer manually.",
+            "minimum_value": "0",
+            "enabled": true,
+            "unit": "\u00b0C"
+        },
+        "machine_nozzle_size":
+        {
+            "default_value": 0.5
+        },
+        "material_diameter": { "default_value": 2.85 }
+    },
+   
+    "settings":
+    {
+                "speed":
+        {
+            "label": "Speed",
+            "icon": "category_speed",
+            "description": "Speed",
+            "type": "category",
+            "children":
+                        {
+                                 "acceleration_enabled":
+                {
+                    "label": "Enable Acceleration Control",
+                    "description": "Enables adjusting the print head acceleration. Increasing the accelerations can reduce printing time at the cost of print quality. Populated values are the firmware defaults.",
+                    "type": "bool",
+                    "default_value": false,
+                    "resolve": "any(extruderValues('acceleration_enabled'))",
+                    "settable_per_mesh": false,
+                    "settable_per_extruder": false
+                },
+             "jerk_enabled":
+                {
+                    "label": "Enable Jerk Control",
+                    "description": "Enables adjusting the jerk of print head when the velocity in the X or Y axis changes. Increasing the jerk can reduce printing time at the cost of print quality. Populated values are the firmware defaults.",
+                    "type": "bool",
+                    "default_value": false,
+                    "resolve": "any(extruderValues('jerk_enabled'))",
+                    "settable_per_mesh": false,
+                    "settable_per_extruder": false
+                }
+             }
+          },
+        "material":
+        {
+            "children":
+            {
+                "material_bed_temperature":
+                {
+                    "children":
+                    {
+                        "material_part_removal_temperature":
```

Comment:
I don't think that this setting is a child of material_bed_temperature.

THe children relationship is used for more specific settings. Eg; "acceleration_wall" is the parent of "acceleration_wall_0" and "acceleration_wall_x". The relationship ensures that if both acceleration_wall_0 and acceleration_wall_x are changed by the user that "acceleration_wall" is disabled (since the engine only looks at the 'leaf' settings)